lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3925,6 +3925,9 @@ namespace {
           }
           if (selfIdx) {
             func->setSelfIndex(selfIdx.value());
+            if (func->getASTContext().LangOpts.hasFeature(
+                    Feature::AddressableParameters))
+              func->getImplicitSelfDecl()->setAddressable();
```

Comment:
This change only handles the `self` parameter, right? Do you account for other parameters elsewhere?

Reply:
Right! For regular parameters it is the last change in `lib/ClangImporter/ImportType.cpp`.